cdap-runtime-ext-dataproc/src/main/java/io/cdap/cdap/runtime/spi/provisioner/dataproc/DataprocClient.java

```diff
@@ -416,6 +419,15 @@ ClusterOperationMetadata createCluster(String name, String imageVersion, Map<Str
         .setPreemptibility(InstanceGroupConfig.Preemptibility.NON_PREEMPTIBLE)
         .setDiskConfig(workerDiskConfig);
 
+      //Set default concurrency settings for fixed cluster
+      if (Strings.isNullOrEmpty(conf.getAutoScalingPolicy())) {
```

Comment:
would be good to put some of the reasoning on the PR comment as code comments here.

Reply:
Sure